CRITICAL - 2025-09-05 13:30:22 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "auth/login.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/login.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('auth/login', [], true)
 3 APPPATH\Controllers\auth.php(12): view('auth/login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE313_HMS_GROUP03\\public\\index.php')
CRITICAL - 2025-09-05 13:30:22 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "auth/login.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/login.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('auth/login', [], true)
 3 APPPATH\Controllers\auth.php(12): view('auth/login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE313_HMS_GROUP03\\public\\index.php')
CRITICAL - 2025-09-05 13:32:14 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:32:14 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:35:51 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "auth/login.php"
[Method: GET, Route: /]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/login.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('auth/login', [], true)
 3 APPPATH\Controllers\auth.php(11): view('auth/login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\ITE313_HMS_GROUP03\\public\\index.php')
CRITICAL - 2025-09-05 13:40:45 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:41:23 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 3.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:41:38 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 3.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:41:52 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:41:53 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:42:43 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:42:43 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:42:44 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 13:42:44 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: /]
in APPPATH\Controllers\auth.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-09-05 14:40:58 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: CLI, Route: db:seed/AdminUserSeeder]
in APPPATH\Models\UserModel.php on line 4.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2025-09-05 15:30:28 --> mysqli_sql_exception: Duplicate column name 'role_id' in C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `us...', 0)
#1 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `us...')
#2 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `us...')
#3 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `us...')
#4 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Migrations\2025-05-28-000001_AddRoleIdToUsers.php(11): CodeIgniter\Database\Forge->addColumn('users', Array)
#5 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(876): App\Database\Migrations\AddRoleIdToUsers->up()
#6 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE313_HMS_GROUP03\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 15:31:13 --> mysqli_sql_exception: Duplicate column name 'role_id' in C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `us...', 0)
#1 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `us...')
#2 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `us...')
#3 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `us...')
#4 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Migrations\2025-05-28-000001_AddRoleIdToUsers.php(20): CodeIgniter\Database\Forge->addColumn('users', Array)
#5 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(876): App\Database\Migrations\AddRoleIdToUsers->up()
#6 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE313_HMS_GROUP03\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 15:33:23 --> mysqli_sql_exception: Duplicate column name 'role_id' in C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `us...', 0)
#1 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `us...')
#2 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `us...')
#3 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `us...')
#4 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Migrations\2025-05-28-000001_AddRoleIdToUsers.php(11): CodeIgniter\Database\Forge->addColumn('users', Array)
#5 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(876): App\Database\Migrations\AddRoleIdToUsers->up()
#6 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE313_HMS_GROUP03\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-09-05 15:36:09 --> mysqli_sql_exception: Duplicate entry 'admin' for key 'username' in C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Seeds\UserSeeder.php(62): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Seeds\DatabaseSeeder.php(12): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Seeder.php(149): App\Database\Seeds\DatabaseSeeder->run()
#9 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\ITE313_HMS_GROUP03\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2025-09-05 15:37:06 --> mysqli_sql_exception: Duplicate entry 'Hospital Administrator' for key 'name' in C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `ro...', 0)
#1 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ro...')
#2 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ro...')
#3 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ro...', NULL, false)
#4 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Seeds\RoleSeeder.php(46): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Seeder.php(149): App\Database\Seeds\RoleSeeder->run()
#7 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Seeds\DatabaseSeeder.php(11): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Seeder.php(149): App\Database\Seeds\DatabaseSeeder->run()
#9 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\ITE313_HMS_GROUP03\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2025-09-05 15:52:00 --> mysqli_sql_exception: Duplicate column name 'role_id' in C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MySQLi\Connection.php(327): mysqli->query('ALTER TABLE `us...', 0)
#1 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `us...')
#2 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `us...')
#3 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `us...')
#4 C:\xampp\htdocs\ITE313_HMS_GROUP03\app\Database\Migrations\2025-05-28-000001_AddRoleIdToUsers.php(11): CodeIgniter\Database\Forge->addColumn('users', Array)
#5 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(876): App\Database\Migrations\AddRoleIdToUsers->up()
#6 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE313_HMS_GROUP03\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE313_HMS_GROUP03\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-09-05 16:13:31 --> Error: Call to undefined function set_value()
[Method: GET, Route: register]
in APPPATH\Views\auth.php\register.php on line 31.
 1 SYSTEMPATH\View\View.php(226): include()
 2 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('auth.php/register', [], true)
 4 APPPATH\Controllers\auth.php(19): view('auth.php/register', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->register()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
